src/components/signatures/signTypedData-variants.js

```diff
@@ -0,0 +1,149 @@
+import {
+  getPermitMsgParams,
+  MSG_PRIMARY_TYPE,
+  splitSig,
+} from '../../signatures/utils';
+import globalContext from '../..';
+
+export function signTypedDataVariantsComponent(parentContainer) {
+  parentContainer.insertAdjacentHTML(
+    'beforeend',
+    `          <div
+            class="col-xl-4 col-lg-6 col-md-12 col-sm-12 col-12 d-flex align-items-stretch"
+          >
+            <div class="card full-width">
+              <div class="card-body">
+                <h4>
+                  Sign Typed Data Variants
+                </h4>
+                <button
+                  class="btn btn-primary btn-lg btn-block mb-3"
+                  id="signBlurOrder"
+                  disabled
+                >
+                  Blur - Order
+                </button>
+                <button
+                  class="btn btn-primary btn-lg btn-block mb-3"
+                  id="signPermitSingle"
+                  disabled
+                >
+                  PermitSingle
+                </button>
+
+                <button
+                  class="btn btn-primary btn-lg btn-block mb-3"
+                  id="signPermitBatch"
+                  disabled
+                >
+                  PermitBatch
+                </button>
+
+                <button
+                  class="btn btn-primary btn-lg btn-block mb-3"
+                  id="signSeaportBulkOrder"
+                  disabled
+                >
+                  Seaport - BulkOrder
+                </button>
+
+                <p class="info-text alert alert-warning">
+                  Result:
+                  <span id="signPermitVariantResult"></span>
+                  <p class="info-text alert alert-warning" id="signPermitVariantResultR">r: </p>
+                  <p class="info-text alert alert-warning" id="signPermitVariantResultS">s: </p>
+                  <p class="info-text alert alert-warning" id="signPermitVariantResultV">v: </p>
+                </p>
+              </div>
+            </div>
+          </div>`,
```

Comment:
Nice new section! Suggestion: could we add a `Verify` section too, so we are sure the wallet works correctly with those signature values?
No need to be in this PR, but would be nice to have this, at some point